### DISCUSSION TITLE
希望安卓端可以同时开启多个串口

### DISCUSSION THREAD

**Discussion by ZeaKyX** [wh201906/SerialTest#36] [Ideas]
如题，测试多串口自发自收需要用到。
如果能支持android的分屏特性那就更好了: )

**Comment by wh201906** [comment]
理论上你可以改包名然后自己编译一遍，这样系统就会识别出两个APP，就能双开了

**Comment by wh201906** [comment]
目前是支持分屏的，可以和其它App一起开

**Comment by ZeaKyX** [comment]
其实希望能是这样的，不过有点得寸进尺了哈哈。https://cloud.tencent.com/developer/article/2145087

**Reply by wh201906** [reply]
学到了，不过这个基于Qt的app可能还是不太好改。如果有多开的需求还是改包名然后手动编译方便一些

**Reply by ZeaKyX** [reply]
是的，qt很麻烦，所以编译都不是简单的事😂如果有现成docker一把梭还好

**Reply by wh201906** [reply]
你可以参考一下它在F-Droid上的构建元数据来编译Android版。我以后可能也会出对应的GitHub Action

https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/priv.wh201906.serialtest.yml
